dubbo-admin-ui/src/components/governance/AccessControl.vue

```diff
@@ -537,7 +541,9 @@ export default {
       this.filter = query['application']
       this.selected = 1
     }
-    this.search()
+    if (this.filter !== null) {
+      this.search()
+    }
   },
   components: {
     Breadcrumb,
```

Comment:
there's no need to add a new method, you can check `this.filter` in `search()`

Reply:
I will notify error msg if this.filter === null in this.search(), but here in mounted(), I just need not to call this.search(), other than notify, so I think this modification is ok